HousingSearchApi/V1/Infrastructure/Sorting/RelatedEntityTypeAsc.cs

```diff
@@ -0,0 +1,16 @@
+using Hackney.Shared.HousingSearch.Gateways.Models.Processes;
+using HousingSearchApi.V1.Interfaces.Sorting;
+using Nest;
+using System.Linq;
+
+namespace HousingSearchApi.V1.Infrastructure.Sorting
+{
+    public class RelatedEntityTypeAsc : ISort<QueryableProcess>
+    {
```

Comment:
I think renaming to PersonNameAsc/Desc will be clearer - especially if we add sorting by address down the line